backends-clickhouse/src/test/scala/org/apache/gluten/execution/GlutenClickHouseDecimalSuite.scala

```diff
@@ -38,7 +38,7 @@ class GlutenClickHouseDecimalSuite
 
   override protected val needCopyParquetToTablePath = true
 
-  override protected val tablesPath: String = basePath + "/tpch-data"
+  override protected val tablesPath: String = basePath + "/tpch-data-sf100"
```

Comment:
the sf of this tpch data is 1G, no 100G

Reply:
Yes this was my mistake.

It's likely SF 0.1 but I am not sure, will confirm before merging.